awx/main/analytics/collectors.py

```diff
@@ -275,11 +275,11 @@ def _copy_table(table, query, path):
                                  main_unifiedjob.job_explanation, 
                                  main_unifiedjob.instance_group_id
                                  FROM main_unifiedjob
-                                 JOIN main_job ON main_unifiedjob.id = main_job.unifiedjob_ptr_id
+                                 LEFT JOIN main_job ON main_unifiedjob.id = main_job.unifiedjob_ptr_id
                                  JOIN django_content_type ON main_unifiedjob.polymorphic_ctype_id = django_content_type.id
-                                 JOIN main_project ON main_project.unifiedjobtemplate_ptr_id = main_job.project_id
-                                 JOIN main_organization ON main_organization.id = main_project.organization_id
-                                 WHERE main_unifiedjob.created > {} 
+                                 LEFT JOIN main_project ON main_project.unifiedjobtemplate_ptr_id = main_job.project_id
+                                 LEFT JOIN main_organization ON main_organization.id = main_project.organization_id
```

Comment:
This join is going to change after @AlanCoding's PR merges.

Reply:
cool, I am fine fixing this another way. Just keep in mind this was basically filtering out everything except jobs. But we want other stuff like `workflowapproval`, `worflowjob`, `projectupdate` etc. , so we'll need the left join, since not everything has org associated

Also @AlanCoding is your PR going to be backported? We need this backported to all versions sending data to analytics, so we get the workflow and other data.

Reply:
no, we do not plan on backporting. that would be very complicated

Reply:
Hm, ok, so can we merge this first? Or it doesn't matter for easier backport?

Reply:
We should wait for @AlanCoding's organization schema to land in `devel` before changing this in `devel`.

If you want to open a PR _like this_ in a backport branch, we'll have to carry a different implementation for that database schema anyways.

Reply:
sounds good, I'll backport and wait for @AlanCoding here